crates/matrix-sdk/build.rs

```diff
@@ -0,0 +1,37 @@
+use std::{env, process};
+
+fn env_is_set(var_name: &str) -> bool {
+    env::var_os(var_name).is_some()
+}
+
+fn ensure(cond: bool, err: &str) {
+    if !cond {
+        eprintln!("error: {err}");
```

Comment:
maybe we just made this a very pretty multi-line thing with some ascii-art? and/or some instructions what to do?